doc/classes/ProjectSettings.xml

```diff
@@ -492,6 +492,9 @@
 		<member name="debug/gdscript/warnings/get_node_default_without_onready" type="int" setter="" getter="" default="2">
 			When set to [code]warn[/code] or [code]error[/code], produces a warning or an error respectively when [method Node.get_node] (or the shorthand [code]$[/code]) is used as default value of a class variable without the [code]@onready[/code] annotation.
 		</member>
+		<member name="debug/gdscript/warnings/implicit_conversion_causes_copy" type="int" setter="" getter="" default="1">
+			When set to [code]warn[/code] or [code]error[/code], produces a warning or an error respectively when an implicit type conversion causes a copy.
```

Comment:
It would be nice to expand the description a little to make it clearer why this might be a problem. The same goes for the warning message.